### PR TITLE
Management tool to retry failed periodic tasks

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@miltontony @jpaidoussi ready for review

File: scheduler/management/commands/trigger_periodic_task.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        if options['timeout']:
+            timeout = options['timeout']
+        else:
+            timeout = None
+
+        def confirm(prompt):
+            if options['confirm']:
+                return True
+            try:
+                return raw_input(
```
Comment (jpaidoussi) [inline scheduler/management/commands/trigger_periodic_task.py]: This won't work on Python 3. Django does include a six util for this though:

from django.utils.six.moves import input

Comment (smn) [reply scheduler/management/commands/trigger_periodic_task.py]: yeah, fixed in a later commit.

**Comment by smn** [comment]
@jpaidoussi happy?

**Review by jpaidoussi — APPROVED** [review]

**Pull request merged**